src/main/java/echoServer/HTTP/HTTPResponse.java

```diff
@@ -0,0 +1,24 @@
+package echoServer.HTTP;
+
+import static echoServer.HTTP.StatusCode.PAGE_NOT_FOUND;
+
+public class HTTPResponse {
+
+    // Int code = new Int();
+    // use enum with values for status code type
+    StatusCode statusCode;
+
+    public HTTPResponse(StatusCode statusCode) {
+        this.statusCode = statusCode;
+    }
+
+
+    public static String buildResponse(StatusCode statusCode){
+        switch(statusCode){
+            case PAGE_NOT_FOUND:
+                return "HTTP/1.0 404 Not Found\n";
```

Comment:
So you should probably pay attention to the comment I made about HTTP specification and CRLF.

Reply:
Also, now that I've reviewed your code, I realized another potential area where you may be running into issues with `http_server_spec` is that you're using `readline()`. Review what an HTTP request looks like!